### PR TITLE
Fix oms3_getElement for components

### PR DESCRIPTION
**Pull request by lochel**
### Purpose

Fix oms3_getElement for components; it was only implemented for systems.

### Type of Change

- Bug fix (non-breaking change which fixes an issue)

### Checklist

- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
-

### CODE REVIEW

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being [tested](https://test.openmodelica.org/hudson/job/OMSimulator-PR/723//console). It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#426

The changes will be pushed if all tests run successfully.

**Comment by OpenModelica-Hudson** [comment]
**[Linux:](https://test.openmodelica.org/hudson/job/OMSimulator-PR/723//console)** Wrong hash: got https://github.com/OpenModelica/OMSimulator/pull/426; expected 492c638da35ffe0430eadaa4c7f9035deaf2feca

File: src/OMSimulatorLib/Scope.cpp [hunk 1]
```diff
@@ -297,20 +294,22 @@ oms_status_enu_t oms3::Scope::getElement(const oms3::ComRef& cref, oms3::Element
     return logError("A model has no element information");
 
   oms3::System* system = model->getSystem(tail);
-  if (!system)
-    return logError("Model \"" + std::string(front) + "\" does not contain system \"" + std::string(tail) + "\"");
+  oms3::Component* component = model->getComponent(tail);
+  if (!system && !element)
```
Comment (adeas31) [inline src/OMSimulatorLib/Scope.cpp]: I think `element` is a typo here.

```suggestion
  if (!system && !component)
```

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being [tested](https://test.openmodelica.org/hudson/job/OMSimulator-PR/726//console). It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#426

The changes will be pushed if all tests run successfully.

**Comment by OpenModelica-Hudson** [comment]
**[Linux:](https://test.openmodelica.org/hudson/job/OMSimulator-PR/726//console)** Wrong hash: got 60a0e6f2d86bfd7053a85ec8ad2764da88879b57; expected 492c638da35ffe0430eadaa4c7f9035deaf2feca

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being [tested](https://test.openmodelica.org/hudson/job/OMSimulator-PR/729//console). It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#426

The changes will be pushed if all tests run successfully.

**Comment by OpenModelica-Hudson** [comment]
All tests were successful. :sun_with_face:

**Pull request closed without merging**